src/commands/compile.ts

```diff
@@ -41,76 +19,25 @@ const command: Command = {
 	hasPermission: () => true,
 	parseArguments: content => /\s*(\S+)\s*([\s\S]+)/g.exec(content)?.splice(1) ?? [],
 	handler: (_, message, compiler, src) => {
-		if (compiler === "langs") {
-			// TODO: future implementation -- switches
-			let result = "";
-
-			// TODO: move this functionality and relevant types to wandbox-api-updated
-			https
-				.get("https://wandbox.org/api/list.json", res => {
-					res.on("data", d => {
-						result += d;
-					});
-					res.on("close", () => {
-						const list: string[] = [];
-						const langToCheck = src.toLowerCase();
-
-						try {
-							const compilerDataList = JSON.parse(result) as Compiler[];
-
-							compilerDataList.forEach(compiler => {
-								if (compiler.language.toLowerCase() === langToCheck) {
-									list.push(`${compiler.language} ${compiler.version}: ${compiler.name}`);
-								}
-							});
-
-							message
-								.reply("Results\n" + list.join("\n").slice(0, 1000))
-								.catch(console.error.bind(console));
-						} catch (e) {
-							console.error(e);
-						}
-					});
-				})
-				.on("error", console.error.bind(console));
-		} else {
-			// TODO: ensure wandbox-api-updated does not throw errors (use a promise instead of a callback?).
-			try {
-				fromStringV2(
-					{
-						compiler: compiler,
-						code: src,
-						save: false, // TODO: Implement at a later date, possibly under a different command.
-					},
-					(err, res) => {
-						const embed = new MessageEmbed()
-							.setTitle("Compile Result")
-							.setFooter(`Compiled with: ${compiler}`);
-
-						if (err) {
-							embed.setColor("#BB3333");
-							embed.setDescription("Compilation failed: errors present.");
-							embed.addField("Error", err.message, true);
-						} else {
-							if (res.compiler_error) {
-								embed.setColor("#D95B18");
-								embed.setDescription("Compilation failed: compiler errors present.");
-								embed.addField("Compiler Error", res.compiler_error, false);
-							} else {
-								embed.setColor("#24BF2F");
-								embed.setDescription("Compilation finished.");
-								embed.addField("Program Message", res.program_message ?? "No output.", false);
-							}
-						}
-
-						message.reply(embed).catch(console.error.bind(console));
-					},
-					undefined
-				);
-			} catch (e) {
-				console.error(e);
-			}
-		}
+		fromString({ compiler: compiler, code: src.length == 0 ? "" : src === "0" ? "" : src, save: false })
+			.then(result => {
+				const embed = new MessageEmbed();
+
+				if (result.compiler_error) {
+					embed.setColor("#D95B18");
+					embed.setDescription("Compilation failed: compiler errors present.");
```

Comment:
The above change will combine compiler and program errors.

```suggestion
					embed.setDescription("Compilation failed: errors present.");
```